src/quote/Quote.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * [[Quote]] constructs a framework for Attesters to make an offer for building a [[Claim]] on a [[CTYPE]] in which it includes a price and other terms & conditions upon which a claimer can agree.
+ *
+ * A [[Quote]] object represents a legal **offer** for the closure of a contract attesting a [[Claim]] from the [[CTYPE]] specified within the offer.
+ *
+ * A [[Quote]] comes with a versionable spec, allowing different [[Quote]] specs to exist over time and tracks under which [[Quote]] a contract was closed.
+ *
+ * @module Quote
+ * @preferred
+ */
+
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+
+import Ajv from 'ajv'
+import QuoteSchema from './QuoteSchema'
+import Identity from '../identity/Identity'
+import IQuote, { IQuoteAgreement, IQuoteAttesterSigned } from '../types/Quote'
+import { hashObjectAsStr, verify } from '../crypto/Crypto'
+
+export function validateQuoteSchema(
+  schema: object,
+  validate: object,
+  messages?: string[]
+): boolean {
+  const ajv = new Ajv()
+  ajv.addMetaSchema(QuoteSchema)
+  const result = ajv.validate(
+    JSON.parse(JSON.stringify(schema)),
+    JSON.parse(JSON.stringify(validate))
```

Comment:
why are you cloning it that way here?

Reply:
It returns false. 

I don't know why this is necessary, but it is. I guess it was what was discussed live.